src/tiffcomposite_int.cpp

```diff
@@ -24,22 +24,22 @@
  */
 // *****************************************************************************
 // included header files
-#include "config.h"
+#include "tiffcomposite_int.hpp"
```

Comment:
This one I like it. It seems like clang-tidy it is able to find the pair .h of the .cpp and place it on top :clap: